File: aoj/volume100/csharp/A10011.cs
using System;
using System.Linq;

class AOJ
{
	static void Main()
	{
		Console.ReadLine(); // Skip 'n'
		var a = Console.ReadLine()
			.Split()
			.Reverse()
			.ToArray();

		Console.WriteLine(string.Join(" ", a));
	}
}

File: aoj/volume100/scala/A10011.scala
object Main {
  def main(args: Array[String]) {
    scala.io.StdIn.readLine // to skip n
    val a = (scala.io.StdIn.readLine split " ").map(_.toInt)
    val reversed = a.reverse
    println(reversed.mkString(" "))
  }
}
